app/assets/locales/android_translatable_strings.txt

```diff
@@ -19,6 +19,16 @@ updates.pinned.download=Downloading updates
 updates.pinned.progress=${0}/${1}
 install.current.version=Current version: ${0}
 
+updates.available.title=Updates Available
+update.required.title=Update Required
+update.later.option=I'll update later.
+prompted.apk.update.info=There is a newer version of CommCare available, and it is recommended that you update now.
+forced.apk.update.info=There is a newer version of CommCare available. You must update before continuing to use CommCare.

```

Comment:
Is what on our radar? Right now I just didn't design the non-forced updates to have any date associated with them, since I wasn't sure how we'd use it. I can add that if we want it just as something to display to the user in this message, but I was envisioning that by the time app builders would usually be setting an update prompt, they would most likely just want it to be done asap.